src/bin/server.rs

```diff
@@ -15,16 +14,18 @@ use self::codedaily_backend::middleware::Auth;
 use self::codedaily_backend::helpers::*;
 use codedaily_backend::schema::links::dsl::*;
 use codedaily_backend::schema::users::dsl::*;
+use codedaily_backend::schema::comments::dsl::*;
 use self::diesel::prelude::*;
-use self::diesel::associations::HasTable;
 use rocket_contrib::{Json, Value};
 use std::io;
+use std::cmp::max;
 use std::path::{Path, PathBuf};
 use rocket::response::NamedFile;
 use crypto::digest::Digest;
 use crypto::sha2::Sha256;
 
 const LINKS_PER_PAGE: i64 = 30;
+const COMMENTS_PER_PAGE: u32 = 30;
```

Comment:
Do you think we should move this constant (and the `LINKS_PER_PAGE` as well) into a configurable file, so we can change it  anytime?

Reply:
Might be interesting. I'm not sure these values would change all that much though (changing enough that recompiling would be a bother). Usually this is more about fine tuning for performance and once it's settled it doesn't change.

That said we could just do it for the sake of doing it :wink: